spec/lib/common/client/error_handler_spec.rb

```diff
@@ -16,69 +15,74 @@
 
   describe '#log_error' do
     subject { described_class.new(client_error.new) }
-    
-    it 'calls instance log error method' do
+
+    xit 'calls instance log error method' do
       allow_any_instance_of(described_class).to receive(:log_error).and_return(true)
       expect(subject.log_error).to be(true)
     end
-    
-    it 'calls the super log error' do
+
+    xit 'calls the super log error' do
       allow_any_instance_of(SentryLogging).to receive(:log_error).and_return(true)
       expect(subject.log_error).to be(true)
     end
   end
 
   describe '#transformed_error' do
-  
     context 'when given an error without a specific counterpart' do
       subject { described_class.new(StandardError.new).transformed_error }
-      it 'returns the correct default error' do
+
+      xit 'returns the correct default error' do
```

Comment:
Ah, these tests might not be applicable any longer after the rescue block breakout in the application controller